src/Microsoft.TemplateEngine.Core.Contracts/IGlobalRunConfig.cs

```diff
@@ -15,5 +15,7 @@ public interface IGlobalRunConfig
         IReadOnlyList<IReplacementTokens> Replacements { get; }
 
         IReadOnlyList<ICustomOperationModel> CustomOperations { get; }
+
+        IReadOnlyList<IReplacementTokens> FileNameReplacements { get;}
```

Comment:
To avoid modifying core contract, potentially `FileNameReplacements` can be kept private to `SimpleConfigModel`